samples/sample_common/include/brc_routines.h

```diff
@@ -245,12 +245,13 @@ class AVGBitrate
         {
             if (bPanic || bSH)
             {
-                m_maxWinBitsLim = mfx::clamp((GetLastFrameBits(windowSize, false) + m_maxWinBits)/2, GetMaxWinBitsLim(), m_maxWinBits);
+                m_maxWinBitsLim = std::max(std::min((GetLastFrameBits(windowSize, false) + m_maxWinBits)/2, m_maxWinBits), GetMaxWinBitsLim());
             }
```

Comment:
Clamp returns `std::min(m_maxWinBits, std::max(GetLastFrameBits(windowSize, false) + m_maxWinBits)/2, GetMaxWinBitsLim())`, but before we used to get `IPP_MAX(IPP_MIN(GetLastFrameBits(windowSize, false) + m_maxWinBits)/2, m_maxWinBits), GetMaxWinBitsLim())`
[0b43390](https://github.com/Intel-Media-SDK/MediaSDK/pull/1647/commits/0b43390c8d268c58790d7f257fceed0b4dc38565)

Reply:
Hi @egorovdanil . I consulted with @takulako before this change. What exactly is the issue? I understand that code change here is not so straightforward, but anyway, min + max means clamp. So if you think that clamp is incorrect, you can rearrange parameters of the clamp function instead

Reply:
Hello, @mgonchar. Yes, the clamp is correct, the error was in the wrong brackets in the clamp function call below :)

Reply:
yep, I rechecked the original code. You are right :)